content/articles/1912-announce-tutorial.md

```diff
@@ -0,0 +1,18 @@
+Title: Sciwork 2020 tutorials
+Date: 2019-12-22 17:00
+Category: General
+
+In the coming sciwork conference, we are planning for two hands-on tutorials to
+introduce important programming skills to the audience:
+
+- [Hypothesis Testing with Python](../../tutorial/hypothesis-testing.html)
```

Comment:
I don't like the relative link, but for some reason that I don't know, https://docs.getpelican.com/en/stable/content.html#ref-linking-to-internal-content doesn't work for me, so I am living with this workaround.